src/tsv.js

```diff
@@ -72,8 +74,28 @@ define([
         return rows;
     }
 
+    function escape(value) {
+        if (value === null || _.isUndefined(value)) {
```

Comment:
I had originally written `if (!value)`, but that converts `0` to an empty string which is almost certainly not desired (see [the test for that](https://github.com/dimagi/Vellum/blob/155189-fix-tsv-encoding/tests/tsv.js#L101)). This is meant to handle the things you'd likely want to serialize in a TSV file, which includes strings and numbers. `false` falls into the category of stuff that will probably just not do the right thing if you serialize it. Objects and arrays also fall into that category.